doc/supportedPlatformVerions.md

```diff
@@ -1,78 +1,49 @@
 # Supported platforms and versions
 
-## Supported stretch versions
+## dotnet
 
-### dotnet
-
-- 1.1.14
-- 2.1.801
-- 2.1.802
```

Comment:
@kumaramit-msft please modify this file where necessary and omit all other auto-generated changes

Reply:
Removed changes to this file, not sure if any of those change was needed. But Please let me know if some changes has to be kept here. Basing my judgement on earlier changes to this tool (https://github.com/microsoft/Oryx/commit/c256254c27d9114e579c308f3d1fa41eb6bb3f3e)